WEBKIT-MEDIA-STUBS.md

```diff
@@ -0,0 +1,53 @@
+# Web Testing
+
+The web project needs WebKit and Media shared libraries to run tests.
+
+These can be supplied in a number of ways. See sections below.
+
+## Compiled from source
+
+Specify these Gradle properties to enable building of WebKit and Media libraries from source:
+
+    -PCOMPILE_WEBKIT=true -PCOMPILE_MEDIA=true
+
+Note that these take some time to build.
+
+If you are not actively working on these sources, you may want to cache the output by copying it to one of the folders mentioned below.
+
+
+## Cached libraries
+
+You can place WebKit and Media shared libraries in these folders:
+
+* Unix libraries (*.so or *.dynlib files)
+````
+    $projectDir/../caches/sdk/lib
```

Comment:
I would just list this one and not the other two. In particular, I don't want to encourage developers to copy these libraries into the JDK since it could cause unexpected results if those libraries were loaded at runtime.